tools/app/cypress/e2e/app.cy.ts

```diff
@@ -89,19 +150,124 @@ describe('Modify project', () => {
     cy.get('.doc').get('img').get('[alt="cyberismo"]'); // Check that image is present in asciidoc content
   });
 
-  it('creates a new card to project and deletes it', () => {
-    cy.visit('http://localhost:3000/cards');
+  it('select card statuses', () => {
+    cy.get('p').contains('Untitled page').click(); // Navigate to Untitled page in tree menu
+    cy.get('h1').contains('Untitled page');
 
-    cy.get('[data-cy="createNewCardButton"]').click();
-    cy.get('.templateCard').contains('Decision').click();
-    cy.get('[data-cy="confirmCreateButton"]').click();
+    cy.get('p').contains('Updated title').click(); // Navigate to Updated title in tree menu
+    cy.get('h1').contains('Updated title');
+    // checks trough all possible statuses by clicking trough them in dropdown menu
+
+    cy.get('.MuiMenuButton-variantSoft').contains('Status: Draft').click(); // Verifies text in status button and clicks it
+    cy.get('[role="menu"]').contains('Reopen'); // options in dropdown menu
+    cy.get('[role="menu"]').contains('Archive');
+    cy.get('[role="menu"]').contains('Approve').click(); // Select Approve status
+
+    cy.get('.MuiMenuButton-variantSoft').contains('Status: Approved').click();
+    cy.get('[role="menu"]').contains('Reopen');
+    cy.get('[role="menu"]').contains('Archive').click();
 
-    cy.get('p').contains('Untitled'); // Tree menu item
+    cy.get('.MuiMenuButton-variantSoft').contains('Status: Deprecated').click();
+  });
+
+  it('Add a link between two cards', () => {
+    cy.get('p').contains('Untitled page').click(); // Navigate to Untitled page in tree menu
+    cy.get('h1').contains('Untitled page');
+
+    cy.get('p').contains('Updated title').click(); // Navigate to Updated title in tree menu
+    cy.get('h1').contains('Updated title');
+  
+    cy.get('.MuiIconButton-root').click(); // Click link button
+    cy.get('p').contains('Linked cards'); // Verifies Linked cards text in page
+    cy.get('.MuiSelect-button').contains('Select Link Type').click(); // Click select link type button
+    cy.get('.Mui-expanded').contains('blocks').click(); // Select blocks
+
+    cy.get('.MuiSelect-button').contains('blocks'); // checks Select Link Type text changed to previously selected blocks option
+    cy.get('.MuiAutocomplete-root').get('[placeholder="Search card"]').click(); // click Search card field
+    cy.get('.MuiAutocomplete-option').contains('Untitled page').click(); // Select Untitled page
+    cy.get('button').contains('Add link').click(); // Click add link button
+
+    cy.get('.css-qmy8nh-JoyStack-root > .MuiStack-root > .MuiTypography-body-sm').contains('blocks');
+    cy.get('.css-qmy8nh-JoyStack-root > .MuiStack-root > .MuiTypography-title-sm').contains('Untitled page');
```

Comment:
Referring to UI elements this way breaks tests easily in the future if we do layout small layout changes etc. I would suggest to use the custom data-cy tag (as is done in some cases already) to mark the elements we want to interact with.